### ISSUE TITLE
Adding Dutch NWB dataset

### ISSUE THREAD

**Issue by RobinLinde** [osmbe/road-completion#87]
After the Open Belgium session yesterday, I started experimenting with usage of the Dutch [NWB](https://nationaalwegenbestand.nl/) dataset. I have a sort-of working version in my [netherlands-dev](https://github.com/RobinLinde/road-completion/tree/netherlands-dev) branch. It does need some more work, but [here](https://gist.github.com/RobinLinde/b4efc91bc9a4a7370e6d08ff3e240c91)'s is the last output I was able to generate. I'm currently looking into filtering the data a bit better, but not everything seems to be documented.

**Comment by jbelien** [comment]
Awesome @RobinLinde 👍 

> I'm currently looking into filtering the data a bit better, but not everything seems to be documented.

Always ready to improve the process and documentation!
What information do you need/miss ?

**Comment by RobinLinde** [comment]
> What information do you need/miss ?

It was mainly missing documentation on the NWB side, in the version I got some road types were undocumented, but I've now figured most of them out, and have [added](https://github.com/RobinLinde/road-completion/commit/2de03edf6a16020887401764f7e82b948b42c7a4) most of them.
I haven't yet figured out how to determine motorways and trunk roads. Theoretically it might be possible by combining several data fields, but I don't think it's that important anyway. For now I'll look in adding a workflow for updates, and will open a PR then.
